draft-ietf-dnsop-svcb-https.md

```diff
@@ -1720,19 +1734,17 @@ of three `item`s:
 This table serves as a non-normative summary of the HTTP mapping for SVCB
 ({{https}}).  Future protocol mappings may provide a similar summary table.
 
-|                                  |                                          |
-| -------------------------------- | ---------------------------------------- |
-| **Mapped scheme**                | "https"                                  |
-| **Other affected schemes**       | "http", "wss", "ws", (other HTTP-based)  |
-| **RR type**                      | HTTPS (65)                               |
-| **Name prefix**                  | None for port 443, else `_$PORT._https`  |
-| **Automatically Mandatory Keys** | `port`, `no-default-alpn`                |
-| **SvcParam defaults**            | `alpn`: \["http/1.1"\]                   |
-| **Special behaviors**            | HTTP to HTTPS upgrade                    |
-
-This table does not indicate any SvcParamKeys that servers are required to
-publish, or that clients are required to implement, because there are none
-in this mapping.
+|                                          |                                         |
+| ---------------------------------------- | --------------------------------------- |
+| **Mapped scheme**                        | "https"                                 |
+| **Other affected schemes**               | "http", "wss", "ws", (other HTTP-based) |
+| **RR type**                              | HTTPS (65)                              |
+| **Name prefix**                          | None for port 443, else `_$PORT._https` |
+| **Automatically Mandatory Keys**         | `port`, `no-default-alpn`               |
+| **SvcParam defaults**                    | `alpn`: \["http/1.1"\]                  |
+| **Special behaviors**                    | HTTP to HTTPS upgrade                   |
+| **Keys that clients must implement**     | None (if HTTP/1.1, default port only)   |
```

Comment:
Maybe add "mandatory" here?

Reply:
That suggests a different type (which didn't specify "mandatory" here) would make "mandatory" an optional-to-implement param.  Which is... weird, verging on nonsensical.

In fact, I think this line is duplicative with "automatically mandatory keys."  Mandatory keys aren't required to be present in every record; a record is unusable if a mandatory type is present and not understood.  If a client MUST implement a key to be compliant with an RR type definition, how is that distinct from the set of automatically mandatory keys?

Reply:
OK, removed.  I agree, this can be inferred from the other rows.